README.md

```diff
@@ -163,6 +163,36 @@ Firmware can also be signed while publishing:
 mix nerves_hub.firmware publish --key test
 ```
 
+### Conditionally applying updates
+
+Applying an update right when it is published is not always a perfect strategy.
+NervesHub allows a custom `UpdateHandler` for this.
```

Comment:
I think your readme needs some updates for the rename

Reply:
:+1:  I'm updating that right now.